interface-impl-amf/src/test/resources/org/mule/amf/impl/10-query-string/api.raml

```diff
@@ -19,6 +19,32 @@ types:
       to: array
       body: string
 
+  cat:
+    type: object
+    properties:
+      name: string
+      color: string
+    additionalProperties: false
+  dog:
+    type: object
+    properties:
+      name: string
+      fangs: string
+    additionalProperties: false
+  parrot:
+    type: object
+    properties:
+      name: string
+      wings: number
+    additionalProperties: false
+  hasHome:
```

Comment:
Is this one being used?

Reply:
Yes, [here](https://github.com/mulesoft/apikit-rest-parser-wrapper/pull/465/files#diff-b4c3ed3478e5b48a220cf8c4468eca706a006658eb20a3277abe0237d74ecfeaR98-R119)